src/features/feature1.js

```diff
@@ -0,0 +1,3 @@
+export default function feature1(a, b) {
```

Comment:
Should not be named as base feature. Instead should be named after its function
```suggestion
export default function add(a, b) {
```